pages/resources/channel.mdx

```diff
@@ -60,41 +60,45 @@ Type `10`, `11` and `12` are only available in API v9 and above.
-| 1 << 10 | MEDIA_CHANNEL                              | This forum channel is a media channel                                                                  |
-| 1 << 11 | SUMMARIES_DISABLED                         | This guild channel has summaries disabled                                                              |
-| 1 << 12 | APPLICATION_SHELF_CONSENT                  | This private channel's recipients consented to the application shelf                                   |
-| 1 << 12 | ROLE_SUBSCRIPTION_TEMPLATE_PREVIEW_CHANNEL | This role subscription tier for this guild channel has not been published yet                          |
+| Value   | Name                                          | Description                                                                                            |
+| ------- | --------------------------------------------- | ------------------------------------------------------------------------------------------------------ |
+| 1 << 0  | GUILD_FEED_REMOVED                            | This guild channel has been removed from the guild's feed                                              |
+| 1 << 1  | PINNED                                        | This thread is pinned to the top of its parent forum channel                                           |
+| 1 << 2  | ACTIVE_CHANNELS_REMOVED                       | This guild channel has been removed from the guild's active channels                                   |
+| 1 << 4  | REQUIRE_TAG                                   | This forum channel requires a tag to create threads in                                                 |
+| 1 << 5  | IS_SPAM                                       | This channel is marked as spam                                                                         |
+| 1 << 7  | IS_GUILD_RESOURCE_CHANNEL                     | This guild channel is used as a read-only resource for onboarding and is not shown in the channel list |
+| 1 << 8  | CLYDE_AI                                      | This thread is created by Clyde AI, which has full access to all message content                       |
+| 1 << 9  | IS_SCHEDULED_FOR_DELETION                     | This guild channel is scheduled for deletion and is not shown in the UI                                |
+| 1 << 10 | IS_MEDIA_CHANNEL                              | This forum channel is a media channel                                                                  |
+| 1 << 11 | SUMMARIES_DISABLED                            | This guild channel has summaries disabled                                                              |
+| 1 << 12 | APPLICATION_SHELF_CONSENT                     | This private channel's recipients consented to the application shelf                                   |
+| 1 << 12 | IS_ROLE_SUBSCRIPTION_TEMPLATE_PREVIEW_CHANNEL | This role subscription tier for this guild channel has not been published yet                          |
```

Comment:
We have decided to not include ``IS_`` in names of constants

Reply:
The `IS_` is part of the actual client code, however.